packages/web-components/src/login/login-with-reapit.svelte

```diff
@@ -0,0 +1,42 @@
+<script>
+  export let clientId
+  export let redirectUri
+</script>
+
+<style>
+  .btn--login {
```

Comment:
Does this output a CSS sheet or does it inline the styles?

Reply:
No css file generated. It just create a <style> tag inside <head/> tag